doc/source/contributing_code.rst

```diff
@@ -699,6 +699,44 @@ A typical function docstring looks like the following:
     
         See `Stackoverflow: Mathjax expression in sphinx python not rendering correctly <http://stackoverflow.com/questions/16468397/mathjax-expression-in-sphinx-python-not-rendering-correclty">`_ for further discussion.
 
+
+-------------------
+Documenting changes
+-------------------
+
+.. _versionadded: https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-versionadded
+.. _versionchanged: https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-versionchanged
+.. _deprecated: https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-deprecated
+
+We use reST constructs to annotate *additions*, *changes*, *deprecations* to the code so that users can quickly learn from the documentation in which version of MDAnalysis the feature is available.
```

Comment:
```suggestion
We use reST constructs to annotate *additions*, *changes*, and *deprecations* to the code so that users can quickly learn from the documentation in which version of MDAnalysis the feature is available.
```